source/releases/lts.html.erb

```diff
@@ -0,0 +1,105 @@
+        their way through the beta process, but you are rewarded with semver guarantees.
+      </p>
+
+      <p>While this provides all the flexibility you need to make stability vs. features tradeoff, there is another orthogonal dimension: how often you can schedule time to upgrade. LTS releases give the community an alternative, sanctioned schedule that works
+        better for users who prefer a slower pace.</p>
+
+      <p>By synchronizing the timing that these users upgrade, the community can decide to focus energy on specific versions, rather than a scattershot attempt to support every possible combination. This should result in more consistent support and easier
+        upgrades for users on the LTS channel.</p>
+
+      <p>The Ember 2.4 branch will be moved into the LTS release channel six weeks after Ember 2.4.0. In other words, Ember 2.4 LTS will ship at the same time as Ember 2.5.0. This means that features begin on Canary, spend six weeks on beta before making it
+        into the stable channel, and spend another six weeks on the stable channel before making it into the LTS release channel.</p>
+
+      <p>This same process repeats every four releases, meaning that the next release on the LTS channel will be Ember 2.8 LTS, which will ship at the same time as Ember 2.9.0.</p>
+
+      <p>For example, the expected release of Ember 2.4 stable (currently in beta) is on February 26th. To give you a sense for the cadence of our release channels, the projected release timeline is as follows.</p>
+
+      <p><img src="/images/blog/2016-02/Releases@3x-35aa8db3.svg" alt="LTS release timeline" class="release-timeline"></p>
+
+      <p><strong>LTS releases will receive critical bugfixes for 6 release cycles (36 weeks).</strong> This means we will support Ember 2.4 LTS with critical bugfixes until Ember 2.10.0 is released, around November 4, 2016.</p>
+
+      <p><strong>LTS releases will receive security patches for 10 release cycles (60 weeks).</strong> This means we will support Ember 2.4 LTS with security patches at least until Ember 2.14.0 is released, around April 21, 2017.</p>
+
+      <p>As you can see in the above diagram, Ember 2.8 LTS is projected to ship around September 23, 2016. This gives you plenty of time to complete the migration to the next LTS release once it comes out. As always, because Ember 2.8 is semver compatible
+        with Ember 2.4, the upgrade should be relatively smooth.</p>
+
+      <p>By the time 2.8 rolls around in roughly six months, any addons that make use of the private <code>view</code> APIs should have had a chance to update.</p>
+    </section>
+    <section>
+      <h3 class="anchorable-toc" id="toc_the-lts-guarantees"><a class="toc-anchor" href="#toc_the-lts-guarantees"></a>The LTS Guarantees</h3>
+      <p>What makes an LTS release different than a stable Ember release? LTS releases come with guarantees about API stability and security:</p>
+      <ul>
+        <li>
+          <p>
```

Comment:
A paragraph element shouldn't be necessary inside of a list element. If you remove it, does the page still look the same?

Reply:
It is used for margin.

Reply:
I'd rather have the padding on the li than add extra markup, but okay.